clap_derive/src/attrs.rs

```diff
@@ -170,11 +169,10 @@ impl Attrs {
 
                 res.kind = Sp::new(Kind::Subcommand(ty), res.kind.span());
             }
-            Kind::Skip(_) => (),
+            Kind::ExternalSubcommand | Kind::Skip(_) | Kind::Arg(_) => (),
```

Comment:
I think I prefer the specific ones rather than grouping unrelated items because of their current implementation

Reply:
reverted